### PR TITLE
Provide option-6, not option-5, for DNS servers.

### PR DESCRIPTION
**Pull request by yomimono**
Signed-off-by: Mindy Preston <mindy.preston@docker.com>

### CODE REVIEW

**Comment by yomimono** [comment]
Fixes #52 .

**Comment by yomimono** [comment]
Additional corrections to providing the subnet mask and providing the *correct* option for NTP servers.

File: src/hostnet/lib/tcpip_stack.ml [hunk 1]
```diff
@@ -49,12 +49,18 @@ let make ~client_macaddr ~server_macaddr ~peer_ip ~local_ip =
...
   let hyperkit : host = {
     hostname = "hyperkit";
-    options = [ Dhcp_wire.Subnet_mask (Ipaddr.V4.of_string_exn "255.255.255.0") ];
+    options = [
+      Dhcp_wire.Domain_name "local";
+      Dhcp_wire.Routers [ local_ip ];
+      Dhcp_wire.Dns_servers [ local_ip ];
+      Dhcp_wire.Ntp_servers [ local_ip ];
+      Dhcp_wire.Subnet_mask (Ipaddr.V4.Prefix.netmask prefix)
+    ];
```
Comment (hannesm) [inline src/hostnet/lib/tcpip_stack.ml]: instead of repeating them, you could use the variable `options` (bound in line 48) here as well, or?

Comment (yomimono) [inline src/hostnet/lib/tcpip_stack.ml]: Jinx ;)

**Comment by yomimono** [comment]
This *appears* to be doing the right thing now; here's the tcpdump output from the final DHCPACK in a  lease transaction:

```
	    Magic Cookie 0x63825363
	    DHCP-Message Option 53, length 1: ACK
	    Lease-Time Option 51, length 4: 3599
	    RN Option 58, length 4: 1799
	    RB Option 59, length 4: 2879
	    Server-ID Option 54, length 4: 192.168.65.1
	    Vendor-Class Option 60, length 12: "udhcp 1.24.2"
	    Subnet-Mask Option 1, length 4: 255.255.255.248
	    Default-Gateway Option 3, length 4: 192.168.65.1
	    Domain-Name-Server Option 6, length 4: 192.168.65.1
	    Domain-Name Option 15, length 5: "local"
	    BR Option 28, length 4: 192.168.65.7
	    NTP Option 42, length 4: 192.168.65.1
```

**Pull request merged**